manifest.json

```diff
@@ -32,14 +32,14 @@
             {
                 "name": "email",
                 "ask": {
-                    "en": "Choose an admin email(Can be other then the LDAP emails and can be changed after installation)"
+                    "en": "Choose an admin email (can be changed after installation)"
                 },
                 "example": "johndoe@example.com"
             },
             {
                 "name": "port",
                 "ask": {
-                    "en": "Enter a port(default:9000).If the port is occupied by other app change it to any other value."
+                    "en": "Enter a port(default:9000). If the port is occupied by any other app, the installation will chose another one."
```

Comment:
Will the port be changed to other port itself if its occupied? 

> the installation will chose another one.
This line gives the impression that it will change by itself if occupied.